src/CommentCard.jsx

```diff
@@ -1,11 +1,45 @@
-export default function CommentCard({ comment }) {
-  const { body, author, created_at } = comment;
-
-  return (
-    <div className="comment-card">
-      <p>{body}</p>
-      <p>comment by: {author}</p>
-      <p>created at: {created_at.slice(0, 10)}</p>
-    </div>
-  );
+import { useState, useEffect } from "react";
+import deleteCommentByID from "./Utils/deleteCommentByID";
+
+export default function CommentCard({ comment, commentList, setCommentList }) {
+  const [deleteComment, setDeleteComment] = useState(false);
+  const [hasDeleted, setHasDeleted] = useState(false);
+  const { body, author, created_at, comment_id } = comment;
+  const condition = author === "grumpy19";
+
+  useEffect(() => {
+    if (deleteComment === true) {
+      deleteCommentByID(comment_id).then(() => {
+        setDeleteComment(false);
+      });
+    }
+  }, [deleteComment]);
+
+  function handleClickDelete() {
+    setDeleteComment(true);
+    setHasDeleted(true);
+  }
+
+  if (hasDeleted === false) {
+    return (
+      <div className="comment-card">
+        <p>{body}</p>
+        <p>comment by: {author}</p>
+        <p>created at: {created_at.slice(0, 10)}</p>
+        {condition ? (
+          <button
+            onClick={() => {
+              handleClickDelete();
+            }}
+          >
+            delete
+          </button>
+        ) : (
+          <></>
+        )}
+      </div>
+    );
+  } else {
+    return <p>comment deleted</p>;
+  }
```

Comment:
Currently, when you post a comment, the commentList state isn't updated with the new comment (perhaps you received this feedback in a ticket for posting a comment. 
Once you have implemented this, make sure that a newly added comment can be deleted